src/main/scala/net/ripe/rpki/publicationserver/util/SSLHelper.scala

```diff
@@ -0,0 +1,81 @@
+package net.ripe.rpki.publicationserver.util
+
+import java.io.FileInputStream
+import java.security.KeyStore
+
+import akka.http.scaladsl.{ConnectionContext, HttpsConnectionContext}
+import javax.net.ssl._
+import net.ripe.rpki.publicationserver.AppConfig
+import org.slf4j.Logger
+
+import scala.util.Try
+
+class SSLHelper(conf: AppConfig, logger: Logger) {
+  def connectionContext: Try[HttpsConnectionContext] = {
+    val sslContext = Try(SSLContext.getInstance("TLS"))
+    sslContext.foreach(_.init(getKeyManagers, getTrustManagers, null))
+    sslContext.flatMap(ctx => Try(ConnectionContext.https(ctx)))
+  }
+
+  private def getTrustManagers: Array[TrustManager] = {
```

Comment:
Looks rather similar with the block of getting KeyManagers, different type.

Reply:
I moved this code and decided to leave it as is (since it works).

This is all a bit cumbersome because we want to make both optional. If we require that both are present at the same time we could just have used one of the patterns from https://doc.akka.io/docs/akka-http/current/server-side/server-https-support.html